source/includes/opts/disableTelemetry.rst

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Could we also include on the docs that we will **not** be tracking IP addresses, hostnames, usernames, passwords, credentials, data stored in users' MongoDB deployments, or personable identifiable information when sending telemetry to Segment?